packages/okta-auth-js/lib/token.js

```diff
@@ -231,7 +235,8 @@ function handleOAuthResponse(sdk, oauthParams, res, urls) {
     var tokenDict = {};
 
     if (res['access_token']) {
-      tokenDict['token'] = {
+      tokenDict['accessToken'] = {
+        value: res['access_token'],
         accessToken: res['access_token'],
         expiresAt: Number(res['expires_in']) + Math.floor(Date.now()/1000),
         tokenType: res['token_type'],
```

Comment:
nit: why not dot notation for all of these?  (underscore isn't a problem, just '-')